demos/restart_demo.sh

```diff
@@ -45,11 +45,10 @@ install_nbgrader () {
 
     # Update git repository.
     cd "${nbgrader_root}"
-    git checkout master
     git pull
 
     # Install requirements and nbgrader.
-    pip3 install -U -r requirements.txt -e .
+    pip3 install -U .
```

Comment:
I'm curious why `-e` doesn't work... is there any way we can keep the old behavior? It's useful for being able to debug/develop the demos.